packages/server-wallet/src/wallet/store.ts

```diff
@@ -280,6 +280,14 @@ export class Store {
     return {states: vars.map(ss => _.merge(ss, channelConstants)), channelState};
   }
 
+  async getObjectivesByIds(objectiveIds: string[]): Promise<DBObjective[]> {
+    return ObjectiveModel.forIds(objectiveIds, this.knex);
+  }
+
+  async getChannelsByIds(channelIds: string[]): Promise<ChannelState[]> {
```

Comment:
This function is added in this PR, but it seems unrelated to the rest of the PR, and it's not used elsewhere. Is that intentional?

Reply:
Fixed in 7b17570